<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpatsBurnDialog</class>
 <widget class="QDialog" name="SpatsBurnDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spats Burn Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>

     <!-- Burn Amount -->
     <item row="0" column="0">
      <widget class="QLabel" name="labelBurnAmount">
       <property name="text">
        <string>Burn Amount:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="burnAmountSpinBox">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>1000000000</number>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
      </widget>
     </item>

    </layout>
   </item>

   <!-- Error Label -->
   <item>
    <widget class="QLabel" name="errorLabel">
     <property name="text">
      <string>Error: Invalid Input</string>
     </property>
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { color: red; }</string>
     </property>
    </widget>
   </item>

   <!-- Buttons -->
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
